test/ConfigProviderTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DotTest\AnnotatedServices;
+
+use Dot\AnnotatedServices\ConfigProvider;
+use PHPUnit\Framework\TestCase;
+
+class ConfigProviderTest extends TestCase
+{
+    protected array $config;
+
+    protected function setup(): void
+    {
+        $this->config = (new ConfigProvider())();
+    }
+
+    public function testHasDependencies(): void
+    {
+        $this->assertArrayHasKey('dependencies', $this->config);
+    }
+
+    public function testDependenciesHasFactories(): void
+    {
+        $this->assertArrayHasKey('abstract_factories', $this->config['dependencies']);
```

Comment:
Please also make sure that `abstract_factories` contains `AnnotatedServiceAbstractFactory`.